injection/interface.go

```diff
@@ -84,8 +91,9 @@ var (
 type impl struct {
 	m sync.RWMutex
 
-	clients   []ClientInjector
-	factories []InformerFactoryInjector
-	informers []InformerInjector
-	ducks     []DuckFactoryInjector
+	clients           []ClientInjector
+	factories         []InformerFactoryInjector
+	informers         []InformerInjector
+	filteredinformers []FilteredInformersInjector
```

Comment:
bit of a nit, but I think this should probably be camelCase:
```suggestion
	filteredInformers []FilteredInformersInjector
```